metadata-ingestion/src/datahub/ingestion/source/looker/looker_lib_wrapper.py

```diff
@@ -154,6 +154,17 @@ def get_user(self, id_: str, user_fields: str) -> Optional[User]:
         # User not found
         return None
 
+    @lru_cache(maxsize=5000)
+    def all_users(self, user_fields: str) -> Sequence[User]:
+        try:
```

Comment:
Update stats to include `all_user_calls` and add `self.client_stats.all_user_calls += 1` here